src/sentry/integrations/slack/analytics.py

```diff
@@ -26,8 +26,9 @@ class SlackIntegrationNotificationSent(analytics.Event):
         analytics.Attribute("organization_id"),
         analytics.Attribute("project_id", required=False),
         analytics.Attribute("category"),
-        analytics.Attribute("actor_id"),
+        analytics.Attribute("actor_id", required=False),
```

Comment:
FYI https://github.com/getsentry/etl/pull/1120 may need to get revived so that nothing is broken upstream - best I could find at the time, it wasn't being used anywhere though